addons/website/static/src/snippets/s_popup/options.js

```diff
@@ -88,6 +88,18 @@ options.registry.SnippetPopup = options.Class.extend({
         this.$target[0].style.setProperty('background-color', color, 'important');
     },
 
+    /**
+     * @override
+     */
+    updateUI:function() {
+        if (this.$target.children().hasClass('s_popup_size_full')) {
+            this.$el.find('.s_popup_backdrop').hide();
+        } else {
+            this.$el.find('.s_popup_backdrop').show();
+        }
+        return this._super(...arguments);
+    },
+
```

Comment:
I don't understand why we shouldn't be able to change the backdrop color in this case.
cc @sbuhl

Reply:
why not adding 'data-dependencies="!s_popup_size_full"' on the we-checkbox in the s_popup_options template.

Reply:
ok sorry, I thought it was when the video is on background .. When the video is on background there is no this class in the DOM.. So you dont have this issue (video continue playing after the close) when the video is on background ?

Reply:
Go to editor > click full size > save > edit > click on popup > change to other size > click on full again > colorpicker will be visible 

due to this issue, I have to write javascript.

Reply:
try this
```
<we-row string="Backdrop" class="s_popup_backdrop">
    <we-checkbox data-name="popup_backdrop_opt"
                 data-select-class="s_popup_no_backdrop|"
                 data-set-backdrop="true"
                 data-dependencies="!s_popup_size_full"
                 data-no-preview="true"/>
    <we-colorpicker data-dependencies="!s_popup_size_full"
                    data-select-style="true"
                    data-css-property="background-color"
                    data-color-prefix="bg-"
                    data-css-compatible="true"/>
</we-row>
```

Reply:
it works on my side

Reply:
It will work for one case; however, color picker is visible and enable to set background color even though Backdrop is disable

Reply:
> It will work for one case; however, color picker is visible and enable to set background color even though Backdrop is disable

you are right